recipes/r-rmarkdown/meta.yaml

```diff
@@ -16,17 +16,17 @@ build:
 # Suggests: shiny (>= 0.11), testthat, digest
 requirements:
   build:
-    - r
+    - r-base
     - r-catools
-    - r-htmltools >=0.2.4
-    - r-knitr >=1.12.3
+    - r-htmltools >=0.3.5
+    - r-knitr >=1.14
     - r-yaml >=2.1.5
 
```

Comment:
r-rmarkdown 1.4 needs more dependencies as it is shown in the travis output:

`ERROR: dependencies ‘base64enc’, ‘jsonlite’, ‘rprojroot’ are not available for package ‘rmarkdown’`

`conda skeleton` is handy for building new cran pkg, which captures all needed dependencies, as in this case:

`conda skeleton cran rmarkdown`

the resulting recipe skeleton is almost ready for the publication while just needing to add the hash.